app/screens/map/mapSelector.js

```diff
@@ -29,11 +30,33 @@ function postalCodeFilterSelector(state) {
   return state.filters.postalCodes;
 }
 
+const cityUnitsSelector = createSelector(
```

Comment:
Refactored selectors. There were so few shared selectors that I decided to make it just a file instead of a folder. We can later change it to a folder if necessary.